### PR TITLE
Add tests for mesheryctl root config and flags

### PR DESCRIPTION
**Pull request by nishantwrp**
Signed-off-by: Nishant Mittal <mittalnishant14@outlook.com>

**Description**

This PR fixes n/a. These tests detect errors in config loading preventing bugs like: https://github.com/meshery/meshery/issues/5905

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/root_test.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	testConfigPath := filepath.Join(fixturesDirPath, "testConfig.yaml")
+
+	tests := []RootCmdTestInput{
+		{
+			Name: "Gets version",
+			Args: []string{"version"},
+		},
+		{
+			Name:             "view the channel to test if meshconfig is correctly created & loaded",
+			Args:             []string{"system", "channel", "view"},
```
Comment (nishantwrp) [inline mesheryctl/internal/cli/root/root_test.go:45]: Used this command i.e `mesheryctl system channel view` to test configs as it's output is simple and short. Any other better alternatives?

**Review by alphaX86 — COMMENTED** [review]
Ok, this is an integration test and it's great. I though have a suggestion, continuing the convo from slack, it's noticed that both config, tar.gz and manifests are being created... it's ok since they are created via `system` commands. But we need to delete them after the tests get over, since we didn't want to leave a trace.

@nishantwrp could you add the above said requirement into the test? Other than that, this test looks pretty neat to me

**Comment by nishantwrp** [comment]
@alphaX86 updated the code to remove all files it generates while testing

File: mesheryctl/internal/cli/root/root_test.go (hunk 1 above)
Comment (leecalcote) [reply mesheryctl/internal/cli/root/root_test.go:52]: Good one. This is as short and sweet as it gets, I believe.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**